index.html

```diff
@@ -0,0 +1,259 @@
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
+        </svg>
+        </a>
+        <div class="navbar-links">
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="intro-body">
+        <div class="intro-second-body">
+          <div>
+            <div class="intro">
+              <h2>I’m Echezona <br> Glad to see you!</h2>
+            </div>
+            <div class="intro-para">
+              <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="connect">
+          <div>
+            <h3>LET'S CONNECT</h3>
+          </div>
+        </div>
+
+        <div class="icons">
+          <ul>
+            <li><a href=""><img src="image/Vector.png" alt=""></a></li>
+            <li>
+              <a href="">
+                <img src="image/Vector(1).png" alt="">
+                <img src="image/Vector(2).png" alt="">
+              </a>
+            </li>
+            <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(4).png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(5).png" alt=""></a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    
+    <section class="card">
+      <div class="section-odd">
+        <div class="case">
+          <div class="first-section-background">
+            <div class="image">
+              <img src="image/Snapshoot Portfolio.png" alt="">
+            </div>
+            <div class="content">
+              <h2>Tonic</h2>
+              <ul>
+                  <li class="canopy">CANOPY</li>
+                  <li><img src="image/Counter.png" alt=""></li>
+                  <li>Back End Dev</li>
+                  <li><img src="image/Counter.png" alt=""></li>
+                  <li>2015</li>
+              </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <ul>
+                  <li><a href="">html</a></li>
+                  <li><a href="">css</a></li>
+                  <li><a href="">javaScript</a></li>
+              </ul>
+              <button type="button"><a href="">See Project</a></button>
```

Comment:
[OPTIONAL] It is not a good practice to put an "A" tag inside a button, I recommend you remove it